test/test-get-passwd.c

```diff
@@ -38,7 +38,9 @@ TEST_IMPL(get_passwd) {
   ASSERT(pwd.shell == NULL);
 #else
   len = strlen(pwd.shell);
+# ifndef __PASE__
```

Comment:
If we stick with the current behavior, I think this should assert that `len == 0`.

Reply:
Based on [the document ](https://www.ibm.com/support/knowledgecenter/ssw_ibm_i_74/rzajq/rzajqprocsetpaseshellinfo.htm), this value on IBMi PASE could be empty string '' (by default) or a meaningful word (if set manually). So we skip the assert here.